src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunnerPool.cs

```diff
@@ -26,10 +26,21 @@ public VsTestRunnerPool(StrykerOptions options, ProjectInfo projectInfo)
             });
         }
 
+        public int DiscoverTests()
```

Comment:
The name of this method is a bit misleading. It only gets the number of test cases. The current name implies that it does more than that.